skpro/distributions/pareto.py

```diff
@@ -0,0 +1,161 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Pareto probability distribution."""
+
+__author__ = ["sukjingitsit"]
+
+import numpy as np
+import pandas as pd
+
+from skpro.distributions.base import BaseDistribution
+
+
+class Pareto(BaseDistribution):
+    r"""Pareto distribution (skpro native).
+
+    The scale :math:`x_m` is represented by the parameter ``xm``,
+    and the Pareto index (or shape parameter) :math:`\alpha`
+    by the parameter ``alpha``.
+
+    Parameters
+    ----------
+    xm : float or array of float (1D or 2D), must be positive
```

Comment:
xm is as it is on wikipedia, but I think it is confusing since there is another parameter x, below. Shall we name it `scale` instead?

Reply:
Sure, that would be nice. I just used the Wikipedia convention initially as different websites had different names for these two parameters.

Reply:
I have implemented the parameter name change from xm to scale, and updated the docstring as such

Reply:
Have added the CDF to the docstring for clarity